server/channels/api4/elasticsearch.go

```diff
@@ -71,7 +71,8 @@ func purgeElasticsearchIndexes(c *Context, w http.ResponseWriter, r *http.Reques
 		return
```

Comment:
Are we handling this case gracefully on the client? I.e. when `RestrictSystemAdmin` is enabled and we're unable to purge. Should we have some sort of recommended next step documented somewhere? What would that be?

Reply:
Ah great catch! I didn't even know about this feature.

I checked and we don't show the ES setting section if `RestrictSystemAdmin` is enabled.

Reply:
If we don't show the ES setting section at all, should the instructions in the DM be different in that case?

Reply:
Does that work for you?